README.md

```diff
@@ -47,17 +47,16 @@ usage: grub2-theme-preview [-h] [--grub-cfg PATH] [--verbose]
                            PATH
 
 positional arguments:
-  PATH                  Path of theme directory (or PNG/TGA image file) to
-                        preview
+  PATH                  Path of theme directory (or PNG/TGA image file) to preview
 
 optional arguments:
   -h, --help            show this help message and exit
-  --grub-cfg PATH       Path of custom grub.cfg file to use (default:
-                        /boot/grub{2,}/grub.cfg)
+  --grub-cfg PATH       Path of custom grub.cfg file to use (default: /boot/grub{2,}/grub.cfg)
   --verbose             Increase verbosity
   --resolution WxH      Set a custom resolution, e.g. 800x600
-  --timeout SECONDS     Set timeout in whole seconds or -1 to disable
-                        (default: 30 seconds)
+  --timeout SECONDS     Set timeout in whole seconds or -1 to disable (default: 30 seconds)
+  --add TARGET=/SOURCE  make grub2-mkrescue add file(s) from /SOURCE to /TARGET in the rescue image
+                        (can be passed multiple times)
```

Comment:
@Natetronn PS: There is one more thing missing in the so-called "synopsis" section. Here's one way of comparing the readme with the actual output (based on GNU `diff`, GNU sed and Bash's process substitution feature and fur current `master`):
```console
# diff -U0 <(sed -n '42,78p' README.md) <(grub2-theme-preview --help)
--- /dev/fd/63  2021-11-28 15:02:43.575468105 +0100
+++ /dev/fd/62  2021-11-28 15:02:43.573468090 +0100
@@ -2 +2,2 @@
-                           [--resolution WxH] [--timeout SECONDS] [--version]
+                           [--resolution WxH] [--timeout SECONDS]
+                           [--add TARGET=/SOURCE] [--version]
@@ -19,0 +21,3 @@
+  --add TARGET=/SOURCE  make grub2-mkrescue add file(s) from /SOURCE to
+                        /TARGET in the rescue image (can be passed multiple
+                        times)
```

Reply:
PS: Environment variable `COLUMNS` and stdout being a TTY or not seem to control line breaking of argparse, e.g. compare the output of:
- `grub2-theme-preview --help` (with TTY)
- `grub2-theme-preview --help | cat` (without TTY)
- `COLUMNS=70 grub2-theme-preview --help` (with custom `COLUMNS` value)

Reply:
I manually edited it for readability here on Github using their built in editor, not realizing it was directly from --help and that it would end up the way it did. Of course, having a usable PR is to be expected.

Reply:
I see now that I was also missing `[--add TARGET=/SOURCE]` from the usage section. And why taking --help directly from the argparse is a wise move.